### ISSUE TITLE
Printing t_0_par, t_0_kep, and t_0_xi

### ISSUE THREAD

**Issue by rpoleski** [rpoleski/MulensModel#73]
When you call `ModelParameters.__repr__()`, then the values of reference epochs (`t_0_par`, `t_0_kep`, and `t_0_xi` in near future) are not printed. @jenniferyee - do we want to keep it that way?

**Comment by jenniferyee** [comment]
Reference epochs should be printed.

**Comment by rpoleski** [comment]
Here is an example code:
```python
import MulensModel as mm

coordinates = "17:54:14.53 −34:46:40.99"
params = {'t_0': 2454656.4, 'u_0': 0.01, 't_E': 50., 'pi_E_E': 0.1, 'pi_E_N': 0.2}

model_params = mm.ModelParameters(params)
model = mm.Model(params, coords=coordinates)
print(model_params)
print()
print(model)
print()

# Add t_0_par:
params['t_0_par'] = 2454656
model_params = mm.ModelParameters(params)
model = mm.Model(params, coords=coordinates)
print(model_params)
print()
print(model)
```
__First__ possible output:
```python
    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 
coords: 17:54:14.53 -34:46:40.99
default magnification method: point_source

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E t_0_par (HJD)
2454656.40000  0.010000    50.0000   0.20000   0.10000 2454656.40000

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E t_0_par (HJD)
2454656.40000  0.010000    50.0000   0.20000   0.10000 2454656.40000
coords: 17:54:14.53 -34:46:40.99
default magnification method: point_source
```
__Second__ possible output:
```python
    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 
t_0_par is assumed to be equal to t_0

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 
t_0_par is assumed to be equal to t_0
coords: 17:54:14.53 -34:46:40.99
default magnification method: point_source

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 
t_0_par (HJD): 2454656.40000

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 
t_0_par (HJD): 2454656.40000
coords: 17:54:14.53 -34:46:40.99
default magnification method: point_source
```
__Third__ possibility:
```python
    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 
t_0_par is assumed to be equal to t_0
coords: 17:54:14.53 -34:46:40.99
default magnification method: point_source
...
```
(the rest would be the same as in the second case).

@jenniferyee - please indicate which one you prefer. You can edit the answers I've written above.

**Comment by jenniferyee** [comment]
I don't like 1.

The difference between 2 and 3 is just whether or not ModelParameters makes any mention of t_0_par? 

If so, I like 2. t_0_par is stored as part of ModelParameters, so that's where it should be printed.

**Comment by rpoleski** [comment]
__Fourth__ possibility - print(ModelParameters) always returns 2 lines. If `t_0_par` is not provided, then we know it equals `t_0`:
```python
    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 
t_0_par is assumed to be equal to t_0
coords: 17:54:14.53 -34:46:40.99
default magnification method: point_source

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E t_0_par (HJD)
2454656.40000  0.010000    50.0000   0.20000   0.10000 2454656.40000

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 
t_0_par (HJD): 2454656.40000
coords: 17:54:14.53 -34:46:40.99
default magnification method: point_source
```
Agreed?

**Comment by jenniferyee** [comment]
My opinion is to make ModelParameters and Model the same.

> print(ModelParameters)
    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 
t_0_par (HJD): 2454656.40000

> print(Model)
    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E 
2454656.40000  0.010000    50.0000   0.20000   0.10000 
t_0_par (HJD): 2454656.40000
coords: 17:54:14.53 -34:46:40.99
default magnification method: point_source

Also, I changed my mind about t_0_par: we should always just give a value (NOT say that it was assumed to be t_0). This is simpler, and simpler is always better for long-term maintenance.

**Comment by rpoleski** [comment]
> ... simpler is always better for long-term maintenance.

Ok, I followed this advise and thought about it once more. Two lines are simpler than 3 lines with different format in the 3rd line. Hence, the code at the top should result in (__fifth possibility__):

```python
    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E t_0_par (HJD)
2454656.40000  0.010000    50.0000   0.20000   0.10000 2454656.40000

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E t_0_par (HJD)
2454656.40000  0.010000    50.0000   0.20000   0.10000 2454656.40000
coords: 17:54:14.53 -34:46:40.99
default magnification method: point_source

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E t_0_par (HJD)
2454656.40000  0.010000    50.0000   0.20000   0.10000 2454656.40000

    t_0 (HJD)       u_0    t_E (d)    pi_E_N    pi_E_E t_0_par (HJD)
2454656.40000  0.010000    50.0000   0.20000   0.10000 2454656.40000
coords: 17:54:14.53 -34:46:40.99
default magnification method: point_source
```

**Comment by jenniferyee** [comment]
t_0_par and t_0_kep are implemented as of commit a134e5b

t_0_xi is implemented in the xal branch, soon to be merged with master. See commit [544d634](https://github.com/rpoleski/MulensModel/commit/544d634b74013c8b8dc830988457070e3041a953)
